cdap-docs/user-guide/source/data-preparation/directives/parse-timestamp.rst

```diff
@@ -0,0 +1,33 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2018 Cask Data, Inc.
+
+===============
+Parse Timestamp
+===============
+
+The PARSE-TIMESTAMP directive parses column values representing unix timestamp as date.
+
+Syntax
+------
+
+::
+
+    parse-timestamp :<column> '<timeunit>'
+
+The ``<column>`` is the name of the column in the record that has timestamp. The ``<timeunit>`` is timeunit of
+timestamp presented by column.
+
+Usage Notes
+-----------
+
+The PARSE-TIMESTAMP directive will parse a column representing Unix timestamp. The column type can be string or long.
+long or string as date objects. The directive also has optional parameter timeunit which can be seconds, milliseconds
+or microseconds. The default timeunit is milliseconds. If the column is `null` or has already been parsed as a date,
```

Comment:
If the column is `null` or is already the timestamp type, applying this directive is a no-op.